.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,108 @@
+  - type: textarea
+    id: internal_notes
+    attributes:
+      label: Internal Notes (Subtasks, DoD, etc.)
+      description: For your own planning, feel free to use Markdown.
+      placeholder: |
+        "## ✅ Definition of Done (DoD)"
+
+        **Note:** This DoD is meant as a helpful guide; not every item is required for every task or feature. Use what makes sense and skip what doesn't. You're doing great!
+
+          "### 📦 Functionality & Stability"
+          - [ ] All required endpoints and business logic are fully implemented  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)